101-automation-configuration/nested/provisionConfiguration.json

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "automationAccountName": {
+      "type": "string"
+    },
+    "location": {
+      "type": "string"
+    },
+    "adminUsername": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "safeModePassword": {
+      "type": "securestring"
+    },
+    "domainName": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "dscConfigurations": {
+      "DomainControllerConfig": {
+        "name": "DomainControllerConfig",
+        "description": "minimum viable configuration for a domain controller role",
+        "url": "https://raw.githubusercontent.com/mgreenegit/azure-quickstart-templates/101-automation-configuration/101-automation-configuration/dscConfigurations/DomainControllerConfig.ps1",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts

Reply:
removed